app/static/profiles/book.json

```diff
@@ -245,12 +281,29 @@
             {
                 "id": "bfp:Instance:ElectronicBook",
                 "class": {
-                    "classLabel": "Electronic",
+                    "classLabel": "E-book",
                     "id": "http://bibfra.me/vocab/marc/Electronic",
-                    "instantiates": "http://bibfra.me/vocab/marc/Books",
+                    "instanceOf": "http://bibframe.org/vocab/Book",
                     "propertyTemplate": [
+			{
+                            "mandatory": "true",
+			    "property": {
+                            "id": "http://bibfra.me/vocab/lite/instantiates",
```

Comment:
At the moment, this is a special case property, so it isn't needed in the profiles.  You can remove property templates for `instantiates` (and holdings, though I guess that's not really defined yet).